tests/integration_tests/validation/conformance_suite_configurations/edinet.py

```diff
@@ -19,13 +19,18 @@
         "EDINET.EC5700W.GFM.1.8.4": 2,
         "EDINET.EC8023W": 2,
         "EDINET.EC8027W": 1,
+        "EDINET.EC8065W": 1,
+        "EDINET.EC8067W": 1,
```

Comment:
Are these being placed here because of uncertainty about the implementation? That was the original plan for this configuration, although I wasn't consistent about that until recently. If I'm confident that my implementation is correct, I've been putting these into the testcase xml, even for the "valid" filings. If I have doubts, I've been putting them here so it's easier to clean up once there is more clarity around that particular rule.